modules/security/pages/password-policy.adoc

```diff
@@ -1,32 +1,121 @@
 = Password Policy
 :description: Overview of password policy.
 
-You can enable a password policy to mandate the minimal required complexity of all xref:user-access:user-credentials.adoc#_passwords[user passwords] or xref:user-access:access-control-model.adoc#_acl_password[ACL passwords].
+This doc teachs you how to control password complexities.
 
 NOTE: Regardless of whether the password policy is enabled, no password can be an empty string or null.
 
-== Password policy requirements
-TigerGraph offers the following password policy.
-When enabled, all passwords must meet the following requirements:
-
-* Have a length between 8 and 128 characters
-* Have at least 1 upper-case letter and 1 lower-case letter
-* Have at least 1 number
-* Have at least 1 special character: `!`, `"`, `#`, `$`, `%`, `&`, `'`, `(`, `)`, `*`, `+`, `,`, `-`, `.`, `/`, `:`, `;`, `<`, `=`, `>`, `?`, `@`, `[`, `]`, `^`, `_`, ```, `{`, `|`, `}`, `~`
+== Password policy configuration
+TigerGraph offers the admin can config the following password policy:
 
+* Password min length which between 1 to 128 characters
+* Require at least one uppercase character
+* Require at least one lowercase character
+* Require at least One Digit
```

Comment:
Let's maintain a consistent writing style.
Change it to "Require at least one digit."